rpc/swagger/swagger.yaml

```diff
@@ -1261,6 +1261,9 @@ definitions:
         $ref: "#/definitions/BlockId"
       header:
         $ref: "#/definitions/BlockHeader"
+      number_txs:
+        type: number
```

Comment:
doesn't amino encodes everything as strings?

Reply:
yes I believe so, will make the change